dockerfiles/rhiot-image-streams.json

```diff
@@ -0,0 +1,540 @@
+{
```

Comment:
I don't know because we haven't discussed the whole solution yet. Might be obsolete or not.